ui/hooks/src/use-navigation.ts

```diff
@@ -1,10 +1,17 @@
-import { ItemTypes } from '@akashaorg/ui-awf-typings/lib/app-loader';
+import React from 'react';
 import { useQueryClient } from 'react-query';
+
+import { NavigateToParams } from '@akashaorg/ui-awf-typings';
+import { ItemTypes } from '@akashaorg/ui-awf-typings/lib/app-loader';
 import { IContentClickDetails } from '@akashaorg/design-system/lib/components/EntryCard/entry-box';
-import React from 'react';
+
 import { COMMENT_KEY } from './index';
-import { NavigateToParams } from '@akashaorg/ui-awf-typings';
 
+/**
+ * Hook to navigation to entry's page
+ * @param navigateFn - function to handle navigation
+ * @param currentPostId - id of the current post
+ */
 export const useEntryNavigation = (
```

Comment:
Is this being used?

Reply:
yes, the hook is used in multiple places